update store-passwords-using-reversible-encryption.md (#3787)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/security-policy-settings/store-passwords-using-reversible-encryption.md

Disable the **Store password using reversible encryption** policy setting.

>[!Note]

> > After disabling the policy settings, only the new passwords will be forced to be stored using one-way encryption. Existing passwords will be stored using reversible encryption until the password is changed.

Reviewer: ```suggestion
> When policy settings are disabled, only new passwords will be stored using one-way encryption by default. Existing passwords will be stored using reversible encryption until they are changed.
```

> 

Reviewer: ```suggestion
```
